specification/eql/get_status/examples/response/EqlGetStatusResponseExample1.yaml

```diff
@@ -0,0 +1,12 @@
+# summary:
+description: A successful response for getting status information for an async EQL search.
+# type: response
+# response_code: 200
+value: >-
```

Comment:
This is the only place in the `value` field where `>` is used as a separator instead of `|`. I don't know if it's intentional or not, and I'm not even sure if it makes a difference at all, but I thought it might be worth pointing it out.